rfcs/ils-0104-library-kpis.md

```diff
@@ -0,0 +1,424 @@
+- Start Date: 2025-08-01
+- RFC PR: [#104](https://github.com/inveniosoftware/rfcs/pull/104)
+- Authors: Jakob Miesner
+
+# Library KPIs
+
+## Summary
+Add endpoints to InvenioILS for KPIs about the performance of the library and extend Invenio Stats.
+
+## Motivation
+The CERN Library would like to have two dashboards displaying its KPIs for two different user groups:
+ 
+1. Internal Dashboard
+This dashboard is used internally by the librarians to measure the performance of certain processes.
+E.g. when a patron requests a document, how much time passes between the patron issuing the request and the patron receiving the book.
+It allows the librarians to react to find bottlenecks and track the evolution of the library's performance over time. 
+
+2. Outsider/Stakeholder Dashboard
+The Audience of this Dashboard is not the librarians, but rather the patrons and management.
+It displays simpler KPIs that show if the library is working well, while being less detailed and technical than the internal dashboard.
```

Comment:
I am not sure which KPIs would fit in the internal which to external dashboard, was this voiced as a requirement?